File: features/advisor/src/main/java/com/nivasafinance/features/advisor/dto/IdentifierResponse.java
package com.nivasafinance.features.advisor.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class IdentifierResponse {
    private UUID identifier;
}

File: features/identifiers/src/main/java/com/nivasafinance/features/identifiers/dto/IdentifierResponse.kt
package com.nivasafinance.features.identifiers.dto

import java.time.LocalDateTime
import java.util.UUID

data class IdentifierResponse(
    val id: UUID,
    val identifier: String,
    val type: String,
    val extData: Map<String, Any>?,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime,
    val createdBy: String?,
    val updatedBy: String?
)
